config/Makefile.mingw

```diff
@@ -100,6 +100,12 @@ BYTECC=$(TOOLPREF)gcc
 ### Additional compile-time options for $(BYTECC).  (For static linking.)
 BYTECCCOMPOPTS=-O -mms-bitfields -Wall -Wno-unused
 
+### Additional compile-time options for $(BYTECC).  (For debug version.)
+BYTECCDBGCOMPOPTS=-g
+
+### Flag to use to rename object files.  (for debug version.)
+NAME_OBJ_FLAG=-o
```

Comment:
This looks wrong. I think you need a space at the end of the variable's value. Here is how to do it:
```make
EMPTY=
SPACE=$(EMPTY) $(EMPTY)
BYTECCDBGCOMPOPTS=-o$(SPACE)
```
You might want to pull the definitions of EMPTY and SPACE to the beginning of the file.